### ISSUE TITLE
The error occurred when using the getObject function.

### ISSUE THREAD

**Issue by Severusted** [Kong/lua-resty-aws#131]
Hello, when I used the getObject function, I passed in two parameters: Bucket and Key. The status of response was 404. The contents in <BucketName> </BucketName> is my Key. 
I passed the value of Bucket to the Key parameter, and the return value is 200. It can locate the bucket but still cannot find the object because of no keys.
How can I resolve this issue? Or can you provide the source code location for getObject?
Thank you very much!

**Comment by Severusted** [comment]
Sorry, I mean the contents in the label "BucketName" of the response is my Key.

**Comment by Tieske** [comment]
@Severusted did you fix your issue? Some feedback for other users as to how you solved it would be appreciated.

**Comment by Severusted** [comment]
Yes，I fixed this issue in a strange way.
When using  AWS S3 SDK for Java, it needed bucket and fileKey to locate and download files. And the fileKey was the name of file. When using the getObject function of lua-resty-aws, it was customary for me to set the fileKey as the file name. Then I got 404, and the contents in the label "BucketName" of the response is my fileKey(i.e. file name).
The solution was to set the value of fileKey to bucket/fileKey, such as "test-bucket-name/test.txt". Then it could locate and download files.
That' all. Thanks for your work.
